### PR TITLE
MTL-2198

### PR DESCRIPTION
**Pull request by rustydb**
### Summary and Scope

- Fixes: MTL-2198 MTL-2193

#### Issue Type

- Bugfix Pull Request
- RFE Pull Request

Important:

- [fawkes] MTL-2198 New Fawkes LiveCD
- [ansible] New `packages` and `services` role
- [fawkes] Disabled ipv6
- [fawkes] New crucible 0.0.5
- [ansible] New bootserver role for setting up the management-vm's services

General:

- Ansible Lint action change; ansible-community/ansible-lint-action is
  no longer maintained. Using ansible/ansible-lint instead, along with
  newer version.
- (MTL-2197) Added a function for creating a release file
- (MTL-2193) Fixed `[FAILED] Failed to start Remount Root and Kernel File Systems.` on the LiveCD. `systemd-remount-fs.service` was unhappy it could not find `ROOTRAID` but it should be looking for `cow`.
- (MTL-2193) Fixed `[FAILED] Failed to start Rotate log files.` on all boxes, even though I only saw this on the LiveCD and Management-VM. `logrotate.service` was unhappy that some directories it expected from configurations in `/etc/logrotate.d/` did not exist. A new `fix_logrotate_errors` function is invoked in `cleanup.sh`.
- (MTL-2193) Fixed `[FAILED] Failed to start Apply Kernel Variables.` on all nodes. `systemd-sysctl.service` was failing to start due to two invalid settings in `99-tcp.conf` for `net.ipv4.tcp_rmem` and `net.ipv4.tcp_wmem` which had incorrectly formatted values.
- Shrinks base image, shrinks the following items:
  - `/usr/lib/locale` from `215M` down to `4.4M` by removing `glibc-locale` and keeping only `glibc-locale-base`
- Removal of `patterns=base` from `autoinst.template.xml`, no longer forcing the installation of several packages from the base pattern. This has some strange behavior, as "base" will still report as "Installed" but with a significant subset of packages.

New Package Handling:

- Packages and services are now defined in `/vars` and `/group_vars` and are handled by the new "packages" and "services" roles
- Removed all `role/*/vars/{packages,services}` files and their associated tasks, thus removing all duplication of package/service handling
- Packages key off of inventory via `/group_vars`, or they be included in the base runbook through `/vars`
- Packages for Google and Metal are pulled out of common again, defined by `/group_vars/google` and `/group_vars/metal`

### Prerequisites

- [ ] I have included documentation in my PR (or it is not required)
- [x] I have tested this by building an image using this branch HEAD, or this is does not pertain to the node-images pipeline.
- [x] I tested this on internal system or this pertains to the node-images pipeline (if yes and not build related, please include results or a description of the test)
- [ ] I tested this on a vshasta system or this pertains to the node-images pipeline (if yes and not build related, please include results or a description of the test)

### CODE REVIEW

File: group_vars/google/main.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+---
+# DNS Servers to use internally
+# TODO: Do we need to update to HPE internal?
```
Comment (manderson-hpe) [inline group_vars/google/main.yml:26]: Remove a TODO from code?

File: group_vars/ncn/services.suse.yml [hunk 2]
```diff
@@ -24,23 +24,49 @@
...
-    name: kubelet.service
+    name: rasdaemon.service
+    state: stopped
+  - enabled: true
+    name: rollback.service
     state: stopped
   - enabled: false
     name: spire-agent.service
     state: stopped
+# FIXME: Either handle this map, or undo this.
```
Comment (manderson-hpe) [inline group_vars/ncn/services.suse.yml:65]: Make a ticket for this?

**Review by manderson-hpe — APPROVED** [review]

File: group_vars/google/main.yml (hunk 1 above)
Comment (rustydb) [reply group_vars/google/main.yml:26]: I added it for the PR, or for someone else for later.
Leaving it in unless we figure it out before I merge.

File: group_vars/ncn/services.suse.yml (hunk 2 above)
Comment (rustydb) [reply group_vars/ncn/services.suse.yml:65]: This is actually something that should be yanked, it's handled now.

File: pb_node_images_csm.yml [hunk 3]
```diff
@@ -22,8 +22,13 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 #
 ---
-- name: Node Image Compute
+- name: Node Images CSM
   hosts: localhost
```
[the remark this answers is not in the document]
Comment (rustydb) [reply pb_node_images_csm.yml:26]: Yes.

File: group_vars/google/main.yml (hunk 1 above)
Comment (rustydb) [reply group_vars/google/main.yml:26]: @jpdavis-prof , do you know if we can/should update this to be 16.110.135.51 and 16.110.135.52?

Comment (rustydb) [reply group_vars/google/main.yml]: I just started a build that switches it to 16.110.135.51 and 16.110.135.52.. we'll see.

**Review by jpdavis-prof — APPROVED** [review]

File: group_vars/compute/packages.suse.yml [hunk 4]
```diff
@@ -23,7 +23,7 @@
 #
 ---
 packages:
-  - java-11-openjdk-devel=11.0.20.0-150000.3.99.1
+  - java-11-openjdk-devel=11.0.19.0-150000.3.96.1
```
Comment (rustydb) [inline group_vars/compute/packages.suse.yml:26]: This should not be downgrading.

**Pull request merged**